internal/spec/spec.go

```diff
@@ -132,6 +141,12 @@ func (r *RunnerSpec) Validate() error {
 	if r.BootstrapParams.Name == "" {
 		return fmt.Errorf("missing bootstrap params")
 	}
+	if r.Iops != nil && r.VolumeType == "" && r.VolumeType != types.VolumeTypeIo1 && r.VolumeType != types.VolumeTypeIo2 && r.VolumeType != types.VolumeTypeGp3 {
```

Comment:
This condition is a bit strange. If `r.VolumeType == ""`, it is probably already `!= types.VolumeTypeIo1`.

You can probably write this as:

```go
if r.Iops != nil {
    switch r.VolumeType {
    case types.VolumeTypeIo1, types.VolumeTypeIo2, types.VolumeTypeGp3:
    default:
         return fmt.Errorf("EBS iops is only valid for volume types io1, io2 and gp3")
    }
}
```